### DISCUSSION TITLE
Welcome to groupoffice Discussions!

### DISCUSSION THREAD

**Discussion by mschering** [Intermesh/groupoffice#726] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gnggi** [comment]
Hi, I'm new with Groupoffice and want to test it in a configuration suited for our small group. In the past we have been used Kopano wich is no more self hosted useful.
I want to discuss an issue after made a test installation.

**Comment by ausbonn** [comment]
Hi Group-Office Community,

we have been using Tine Groupware within our family of nine users (my wife, two sons, my parents, my sister, my brother and myself) for fifteen years. As the community version is no longer available, I have spent the last couple of years looking for an alternative with ActiveSync support.

Last weekend, we migrated from Tine Groupware (version 2023) to GroupOffice, and the process was straightforward! It is hosted alongside NextCloud on a root server using Docker. I have been maintaining our own mail server for 25 years.

I just wanted to introduce myself.

I am looking forward to 15 years with GroupOffice. But then I will be well past 70.

**Reply by mschering** [reply]
Nice to have you here. Welcome to the Group-Office family!
